core/src/main/java/discord4j/core/object/audit/ChangeKey.java

```diff
@@ -147,6 +147,13 @@
     public static final ChangeKey<Integer> EXPIRE_GRACE_PERIOD = changeKey("expire_grace_period", INTEGER_PARSER);
     /** New user limit in a voice channel */
     public static final ChangeKey<Integer> USER_LIMIT = changeKey("user_limit", INTEGER_PARSER);
+    /** entity type of guild scheduled event changed */
+    public static final ChangeKey<Integer> ENTITY_TYPE = changeKey("entity_type", INTEGER_PARSER);
+    /** channel ID for guild scheduled event changed */
+    public static final ChangeKey<String> LOCATION = changeKey("location", STRING_PARSER);
+    /** status of guild scheduled event changed */
+    public static final ChangeKey<Integer> STATUS = changeKey("status", INTEGER_PARSER);
```

Comment:
not sure about the name of all this maybe can be better a thing like EVENT_STATUS?
and in the same case of STATUS maybe its better not use a Integer and use a enum for the status? https://discord.com/developers/docs/resources/guild-scheduled-event#guild-scheduled-event-object-guild-scheduled-event-status